src/OpenLoco/Paint/PaintMiscEntity.cpp

```diff
@@ -10,152 +10,205 @@ using namespace OpenLoco::Interop;
 
 namespace OpenLoco::Paint
 {
-    // 0x00440325
-    void paintMiscEntity(PaintSession& session, MiscBase* base)
+    static void paintExhausEntity(PaintSession& session, MiscBase* base)
+    {
+        Gfx::drawpixelinfo_t* dpi = session.getContext();
+        if (dpi->zoom_level > 1) // 00440331-0044033C
+        {
+            return;
+        }
+        Exhaust* exhaustObject = base->as_exhaust();
+
+        SteamObject* steamObject = exhaustObject->object(); // 00440342-00440349
+        static_assert(offsetof(SteamObject, var_16) == 0x16);
+        static_assert(offsetof(SteamObject, var_1A) == 0x1A);
+        static_assert(offsetof(SteamObject, sound_effect) == 0x1E);
+
+        uint8_t* edi = (exhaustObject->object_id & 0x80) == 0 ? steamObject->var_16 : steamObject->var_1A; // 00440354-0044035D
+        uint32_t imageId = edi[2 * exhaustObject->var_26];                                                 // 00440350, 00440360
+        imageId = imageId + steamObject->baseImageId + steamObject->var_0A;                                // 00440364-00440367
+
+        if ((steamObject->var_08 & 8) != 0) // 0044036A-00440370
+        {
+            session.addToPlotListAsParent(imageId, { 0, 0, base->z }, { 1, 1, 0 }); // 00440372-00440386
+        }
+        else
+        {
+            session.addToPlotListAsParent(imageId, { 0, 0, base->z }, { 1, 1, 0 }, { -12, -12, base->z }); // 0044038F-004403BC
+        }
+    }
+
+    static void paintRedGreenCurrencyEntity(PaintSession& session, MiscBase* base)
```

Comment:
As far as I know, the currency effects in this game are painted in company colours rather than red/green (like RCT). I think the name should reflect that. Naming things is hard, but perhaps something like
```suggestion
    static void paintCurrencyEffectEntity(PaintSession& session, MiscBase* base)
```